src/gpuarray_buffer_cuda.c

```diff
@@ -147,13 +148,49 @@ static int setup_lib(void) {
         return error_set(global_err, GA_IMPL_ERROR, "cuDriverGetVersion failed");
       major = tmp / 1000;
       minor = (tmp / 10) % 10;
+      #if defined(WIN32) || defined(_WIN32) || defined(WIN64) || defined(_WIN64) || defined(__APPLE__)
```

Comment:
Based on code in libnvrtc.c ( https://github.com/Theano/libgpuarray/blob/master/src/loaders/libnvrtc.c#L49 ), we just look for `libnvrtc.so` on Linux (without explicitely specifying a version).